src/monmove.cpp

```diff
@@ -344,6 +459,35 @@ void monster::plan()
         }
     }
 
+    //Check if steals, must be idle and have no target in sight.
+    if ((friendly >= 0 || target != nullptr) && !fleeing && !has_effect(effect_looting)) {
+        //If stealing monster, inventory is empty and with a bit of luck...we search for loot.
+        if (will_steal && inv.empty() && one_in(100)) {
+            //Find all lootable items within radius
+            auto lootable_items = find_loot_in_radius(pos(), MONSTER_LOOT_DIST);
+
+            if (!lootable_items.empty()) {
+                item_goal = select_desired_loot(lootable_items);
+
+
+                set_dest(item_goal.position());
+                add_effect(effect_looting, 50_turns);
+            }
+            else //TODO: Get frustrated?
```

Comment:
is this a wip here? i wouldn't leave this in for the finished pr.

Reply:
Removed comment, haven't come up with a good idea to implement here yet.